### ISSUE TITLE
Main nav. Improve contrast of links

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#2116]
## Description of issue

macbook pro
Built in maginfication
Chrome

[Fable video](https://app.makeitfable.com/share/685ec98a816251063689d19f?video=1)

Describe what is failing
- blue inks on gray background could have higher contrast

## Potential fix

- Consider aligning designs with GCDS: white background with top and bottom borders on the nav group. 

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

## Acceptance Criteria

- [ ] Should this issue be added to the A11y Statement if it cannot be resolved soon?
